Examples/Swift/ViewController.swift

```diff
@@ -121,7 +133,7 @@ class ViewController: UIViewController, MGLMapViewDelegate {
         
         _ = Directions.shared.calculate(options) { [weak self] (waypoints, routes, error) in
             guard error == nil else {
```

Comment:
While you're at it, let's replace this with a `guard let` so we don't have to force unwrap the error.